### PR TITLE
Fix #907: Faq content highfi

### LINKED ISSUE #907: Highfi and content for FAQSingleActivity
Mock: https://xd.adobe.com/view/0687f00c-695b-437a-79a6-688e7f4f7552-70b6/screen/195a7967-e7f0-4f62-a99c-8fac560e41d8/HL-FAQ-How-can-I-delete-v2-/

Data: https://docs.google.com/spreadsheets/d/1O9B8oZSXAS3RUm8w2JxHmWRHsh4bNHMOYSRCYLRFK84/edit#gid=0

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

This PR fixes the highfi of FAQSingle and adds content too.

## References
https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/195a7967-e7f0-4f62-a99c-8fac560e41d8/specs/
https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/06cf9db7-4825-443f-a82b-c77d95113965/specs/
https://docs.google.com/spreadsheets/d/1O9B8oZSXAS3RUm8w2JxHmWRHsh4bNHMOYSRCYLRFK84/edit#gid=0

## Screenshot
Screenshot_1598252540
Screenshot_1598252545

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/layout-land/faq_single_activity.xml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@id/faq_single_app_bar_layout">
+
+      <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/oppiaGreyBackground"
+        android:clipToPadding="false"
+        android:overScrollMode="never"
+        android:paddingBottom="108dp"
```
Comment (rt4914) [inline app/src/main/res/layout-land/faq_single_activity.xml:47]: The grey background and `108dp` padding is not available in landscape mock but considering that this is the consistent UI that we have followed across many screens I have kept it in this xml file too. You can check `land/faq_list_fragment.xml` file too for reference.

File: app/src/main/res/layout/faq_content.xml [hunk 2]
```diff
@@ -27,10 +27,10 @@
...
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginStart="28dp"
-        android:layout_marginTop="16dp"
-        android:layout_marginEnd="28dp"
-        android:layout_marginBottom="16dp"
+        android:paddingStart="28dp"
+        android:paddingTop="16dp"
+        android:paddingEnd="28dp"
+        android:paddingBottom="16dp"
```
Comment (rt4914) [inline app/src/main/res/layout/faq_content.xml]: Converted margin to padding to provide more clickable area to user.

File: app/src/main/res/layout-land/faq_content.xml [hunk 3]
```diff
@@ -28,11 +28,11 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginStart="76dp"
-        android:layout_marginTop="16dp"
         android:layout_marginEnd="76dp"
-        android:layout_marginBottom="16dp"
         android:fontFamily="sans-serif-medium"
         android:onClick="@{(v) -> viewModel.clickOnFAQQuestion()}"
+        android:paddingTop="16dp"
+        android:paddingBottom="16dp"
```
Comment (rt4914) [inline app/src/main/res/layout-land/faq_content.xml:35]: Converted top/bottom margin to padding to provide more clickable area to user.

File: app/src/main/res/layout-land/faq_single_activity.xml [hunk 4]
```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.story.StoryViewModel" />
```
Comment (anandwana001) [inline app/src/main/res/layout-land/faq_single_activity.xml]: Why we need `StoryViewModel` in faq?

**Review by anandwana001 — APPROVED** [review]
LGTM, Thanks @rt4914

File: app/src/main/res/layout-land/faq_single_activity.xml (hunk 4 above)
Comment (rt4914) [reply app/src/main/res/layout-land/faq_single_activity.xml]: Removed it in both files in latest commit.

Comment (anandwana001) [reply app/src/main/res/layout-land/faq_single_activity.xml]: Thanks, Good to go.

**Pull request merged**